bedrock/products/redirects.py

```diff
@@ -14,4 +14,5 @@
     # Issue 11875
     redirect(r"^vpn/download/windows/?$", "products.vpn.windows-download"),
     redirect(r"^vpn/download/mac/?$", "products.vpn.mac-download"),
+    redirect(r"^products/mozsocial/invite/?$", "mozorg.home"),
```

Comment:
**Suggestion (non-blocking):** would redirecting to `products.landing` be a better fit perhaps?

Bonus points for adding a redirect test.